src/tree_data.c

```diff
@@ -3309,14 +3309,9 @@ lyd_get_or_create_leafref_links_record(const struct lyd_node_term *node, struct
 LIBYANG_API_DEF LY_ERR
 lyd_leafref_get_links(const struct lyd_node_term *node, const struct lyd_leafref_links_rec **record)
 {
-    struct lyd_leafref_links_rec *record2 = NULL;
-    LY_ERR ret;
-
-    assert(record);
-
-    ret = lyd_get_or_create_leafref_links_record(node, &record2, 0);
-    *record = record2;
-    return ret;
+    LY_CHECK_ARG_RET(NULL, node, LY_EINVAL);
+    LY_CHECK_ARG_RET(NULL, record, LY_EINVAL);
```

Comment:
This macro supports up to 6 arguments to check so you can join these lines. And please leave a blank line after it.

Reply:
Done